managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedCursorMXBean.java

```diff
@@ -70,4 +70,33 @@
      */
     long getPersistZookeeperErrors();
 
+    /**
+     * Add write data to a ledger of a cursor (in bytes).
+     *
+     * @param size Size of data written to cursor (in bytes)
+     */
+    void addWriteCursorLedgerSize(long size);
```

Comment:
This method is updating two variables (`writeCursorLedgerSize` and `writeCursorLedgerLogicalSize`). It would be helpful for future readers of this interface to see the Javadoc explain what this method should update. Otherwise, it would be confusing to see a `getWriteCursorLedgerLogicalSize` without an associated `addWriteCursorLedgerLogicalSize` method.

Reply:
Ok, thanks @michaeljmarshall .